modules/post/multi/gather/jenkins_gather.rb

```diff
@@ -319,17 +320,32 @@ def get_key_material(home, platform)
   end
 
   def find_home(platform)
+    if datastore['JENKINS_HOME']
+      if exist?(datastore['JENKINS_HOME'] + '/secret.key.not-so-secret')
```

Comment:
I thought this file doesn't contain any secrets according to https://stackoverflow.com/questions/38465270/wheres-the-encryption-key-stored-in-jenkins and specifically https://github.com/jenkinsci/jenkins/commit/3dc13b957b14cec649036e8dd517f0f9cb21fb04 mentions it was added to note that:

```
indicates that the secret.key is generated by the version of Jenkins post SECURITY-49.
this indicates that there's no need to rewrite secrets on disk
```

Reply:
It doesn't contain secrets and is used purely to discover the installation directory of Jenkins.